### PR TITLE
Variation Electric: Fix button hover content shift issue in

### PR DESCRIPTION
**Pull request by tahmidulkarim**
This fixes the issue: #213

Added border in the default state so in the hover state the border that is added won't cause the contents to shift anymore.

### CODE REVIEW

File: styles/electric.json [hunk 1]
```diff
@@ -34,6 +34,11 @@
 	"styles": {
 		"elements": {
 			"button": {
+				"border": {
+					"style": "solid",
+					"width": "1px",
```
Comment (madhusudhand) [inline styles/electric.json]: 2px is the default for outline button. It is best to define the `2px` border. update the same for hover state.

image

**Review by madhusudhand — COMMENTED** [review]
Thank you for working on this. 
Posted a comment inline for the border thickness.

File: styles/electric.json (hunk 1 above)
Comment (tahmidulkarim) [reply styles/electric.json]: Thank you for the suggestion @madhusudhand. I have update the border thickness.

**Review by madhusudhand — APPROVED** [review]
Pushed a small commit to fix button size issue. The changes are looking good. 👍

**Review by mikachan — APPROVED** [review]
Looks good to me too, thank you!

**Pull request merged**